streamingpro-mlsql/src/main/java/streaming/rest/RestUtils.scala

```diff
@@ -17,6 +17,11 @@ object RestUtils {
        .map{ case (name, value) => new BasicNameValuePair(name, value) }.toSeq
 
     Request.Post(urlString)
+      // Socket timeout is in milliseconds, default to 20 minutes
```

Comment:
It seems like socket/connection timeout should infinity by default, this means  no need to configure this?  

If we hope we can exposure all these config to users, may be we can add a new parameter ? E.G.

```
httpClientPost(urlString: String, data: Map[String, String],config:Map[String,String]=Map())
```